assets/src/js/README.md

```diff
@@ -0,0 +1,56 @@
+# Secure Custom Fields TypeScript
+
+This directory contains TypeScript files for the Secure Custom Fields plugin.
+
+## Experiments
+
+The experiments functionality allows for feature flags in the WordPress admin. This enables gradual rollout of new features and A/B testing.
+
+### Usage
+
+To check if an experiment is enabled in your TypeScript code:
+
+```typescript
+// Check if SCF and experiments are available
+if (typeof window.SCF !== 'undefined' && 
+    typeof window.SCF.experiments !== 'undefined' && 
+    typeof window.SCF.experiments.isEnabled !== 'undefined') {
+  
+  // Check if a specific experiment is enabled
+  if (window.SCF.experiments.isEnabled('experiment_name')) {
+    // Experiment is enabled, implement feature
+  } else {
+    // Experiment is disabled, use default behavior
+  }
+}
+```
+
+### Adding New Experiments
+
+To add a new experiment:
+
+1. Create a new experiment class in `includes/admin/experiments/`
```

Comment:
Is this a "good practices" recommendation or a requirement? What's the reason to create the experiments in this folder instead of creating them in the place they belong, and just registering?

Reply:
Ideally, this way it should be easier to find all experiments registered instead of having to dig into the entire codebase. Also, I'm trying to follow the PHP class based plugin structure, which may differ from how we did things in Gutenberg.

Is more a recommendation than a requirement, but this way all the admin interface will be automatically generated on PHP.

Reply:
Let's remove the readme to prevent external devs to create more experiments.